cdap-docs/reference-manual/source/http-restful-api/lifecycle.rst

```diff
@@ -821,8 +821,7 @@ The request body is a JSON object specifying the details of the schedule to be c
         "type": "<trigger type>",
         ...
       },
-      "timeoutMillis": <timeout in milliseconds>,
-      "scheduleStatus": "<status of the schedule, SUSPENDED or SCHEDULED>"
```

Comment:
why was this removed? Isn't it a part of the response JSON?

Reply:
The title is request body, so I think it should not include the status? Accidentally removed `timeoutMillis`, will put it back